src/GraphQLParser.Tests/Visitors/SDLSorterTests.SortsSchemaDefinition.approved.KitchenSink.graphql

```diff
@@ -139,4 +139,4 @@ extend type Foo {
   seven(argument: [String]): Type
 }
 
-extend type Foo @onType
\ No newline at end of file
+extend type Foo @onType
```

Comment:
This PR fixes these issues -- generated documents did not include a terminating newline before.